source/_components/plugwise.markdown

```diff
@@ -0,0 +1,149 @@
+---
+title: "Plugwise"
+description: "Plugwise Climate integration."
+logo: plugwise.png
+ha_category: Climate
+ha_iot_class: "Local Polling"
+redirect_from:
+  - /components/climate.tado/
```

Comment:
Why a redirect from tado?

Reply:
Should have removed that (was using other climates as reference), my sincere apologies!